content/achievements.md

```diff
@@ -19,6 +19,16 @@ These ones were quite successful:
     <h2 class="timelineMajorMarker"><span>2018</span></h2>
     <dl class="timelineMinor">
 
+      <dt id="event34"><a>RoboCup Junior European Championship</a></dt>
+      <dd class="timelineEvent" id="event34EX" style="display: none; ">
+        <div><ul>
+             <li>Soccer Lightweight - 6th prize</li>
```

Comment:
```suggestion
             <li>Soccer Lightweight - 6th place</li>
```

Reply:
The word "prize" is used in every other achievement, so why should we change it this time?

Reply:
Because we say prize, when we are awarded for the 1st, 2nd or 3rd place.
And I assume that for 6th place we haven't got any award.

Reply:
Ah, OK, I'm sorry for my language knowledge. Thank you for the explanation :)